modules/ROOT/partials/query-context.adoc

```diff
@@ -0,0 +1,12 @@
+// tag::statement[]
+To use the examples on this page, you must set the query context. 
+
+You can set the Query Context from the Query Workbench, {sqlpp} REST API, or cbq shell: 
+
+* To set the query context from the Query Workbench, see the xref:tools/query-workbench.adoc#query-context[Set Query Context] section in Query Workbench.
```

Comment:
Need a colon instead of forward slash for this link:
```suggestion
* To set the query context from the Query Workbench, see the xref:tools:query-workbench.adoc#query-context[Set Query Context] section in Query Workbench.
```